src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -7,6 +7,9 @@ port = {{ port }}
 enable_auth = {{ spec.enable_auth }}
 state_update_notify = True
 state_update_interval_sec = 5
+min_controller_id = {{ spec.min_controller_id }}
+max_controller_id = {{ spec.max_controller_id }}
+enable_spdk_discovery_controller =  {{ spec.enable_spdk_discovery_controller }}
```

Comment:
There is a unit test to make sure the generated nvmeof config is what we expect that needs to be updated if you change the template. Make check won't pass unless the tests passes
```
cephadm/tests/test_services.py::TestNVMEOFService::test_nvmeof_config
```

Reply:
@baum this test still seems to be failing after the most recent push FYI. You can run that test locally with `tox -e py3 -- cephadm/tests/test_services.py::TestNVMEOFService::test_nvmeof_config` run from src/pybind/mgr if you want to check it.

Reply:
@adk3798 thanks for walking me through running the test. Found and got rid of that unnecessary extra whitespace in the template. All good on my end now. Thanks!
```
======================================================== 1 passed, 3 warnings in 0.08s =========================================================
___________________________________________________________________ summary ____________________________________________________________________
  py3: commands succeeded
  congratulations :)
```